qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -429,7 +429,9 @@ <h4>
       <span id="prep-name-span">{{name}}</span> - ID {{prep_id}} ({{data_type}})
       <a class="btn btn-default" data-toggle="modal" data-target="#update-prep-name"><span class="glyphicon glyphicon-pencil"></span> Edit name</a>
       <a class="btn btn-default" href="{% raw qiita_config.portal_dir %}/download/{{download_prep_id}}"><span class="glyphicon glyphicon-download-alt"></span> Prep info</a>
-      <a class="btn btn-default" href="{% raw qiita_config.portal_dir %}/download/{{download_qiime_id}}"><span class="glyphicon glyphicon-download-alt"></span> QIIME map</a>
+      {% if download_qiime_id is not None %}
```

Comment:
This is for backwards compatibility, right? This means that unless a QIIME mapping file has already been created then we won't show this button.

Reply:
That's 100% correct!